docs/src/main/sphinx/release/release-351.md

```diff
@@ -23,7 +23,7 @@
 
 * Publish image as [`trinodb/trino`](https://hub.docker.com/r/trinodb/trino).
 * Change base image to `azul/zulu-openjdk-centos`.
-* Change configuration directories to `/usr/lib/trino/etc` and `/etc/trino`.
+* Change configuration directory to `/etc/trino`.
```

Comment:
Why are you changing release notes for past release?

Reply:
So that they do not misguide users. I am on the fence with this though, I can retract the change.

Reply:
I do not expect anyone will read the source code of release notes. But rather https://trino.io/docs/current/release/release-351.html

Reply:
Of course, the intention would be to republish the release notes (just the same way as we did in 5ae1a1081019de6091fb07c37b068e25f6f6b617), but again I can back it out if you want me to.

Reply:
Oh - I did not know we do that. Fine. Keep the change.

Reply:
We don't generally update the published documentation for a specific release, but release notes are part of the documentation, so any changes for a past release note would be present when the next release is published.

For this change, since we intend to break it in 352, I would want to republish the release notes immediately.